apps/bottle/src/store/query/ServerFetchBoundary.tsx

```diff
@@ -0,0 +1,19 @@
+import { FetchQueryOptions, HydrationBoundary, QueryClient, dehydrate } from '@tanstack/react-query';
```

Comment:
Since `FetchQueryOptions` is a type, it seems better to add 'type' separately! Is there an associated ESLint rule being applied?

```suggestion
import { type FetchQueryOptions, HydrationBoundary, QueryClient, dehydrate } from '@tanstack/react-query';
```

Reply:
What do you think about managing queryClient with useState? It's intended to ensure it's referenced only once in React's lifecycle. I'm curious about your opinion. [document](https://tanstack.com/query/latest/docs/eslint/stable-query-client#rule-details)

Reply:
applied [42efd1f](https://github.com/Nexters/Bottles_Web/pull/38/commits/42efd1f619413219126de2ebc46ddfe3783fab41)
I completely agree. I'm planning to add as a strict convention and a ESLint rule. Thank you for good advice 👍